build.gradle

```diff
@@ -112,7 +114,8 @@ dependencies {
     implementation 'thedarkcolour:kotlinforforge:1.14.0'
     //library 'org.jetbrains:annotations:22.0.0'
     // Chat Age 1.1 - get YAML configs working
-    //library 'com.charleskorn.kaml:kaml:0.38.0'
+    library 'com.charleskorn.kaml:kaml:0.38.0'
+    shade 'com.charleskorn.kaml:kaml:0.38.0'
```

Comment:
Can these not be combined? For example, there's compileOnly and runtimeOnly but implementation covers the two; is there not some way to just make a clean closure that does both in one call? Sorry just being pedantic.

Reply:
My bad, that configuration was removed previously. Let me make sure it's expunged in tip...

This was part of a failed attempt to do said automatic dependency resolution--but it will require someone with more knowledge of Gradle, perhaps, to figure out how to extract a dependency tree _and_ use it in this precise way.